src/nanoemoji/color_glyph.py

```diff
@@ -286,6 +292,15 @@ def _in_glyph_reuse_key(
     )
 
 
+def _intersect(path1: SVGPath, path2: SVGPath) -> bool:
+    # Try computing intersection using pathops; if for whatever reason it fails
+    # (probably some bug) then be on the safe side and assume we do have one...
+    try:
+        return bool(intersection((path1, path2)))
+    except pathops.PathOpsError:
+        return True
```

Comment:
I think we should at least log this; shouldn't really be happening

Reply:
it's really unlikely that we ever encounter this, but I didn't want to risk it given recent breakages with pathops.simplify. I added the logging message

Reply:
> shouldn't really be happening

I just checked and this PathOpsError never happens when building Noto Color Emoji. If we like, we can let the error propagate instead